platforms/android/demo/src/main/java/com/mapzen/tangram/android/MainActivity.java

```diff
@@ -88,6 +189,63 @@ public void onLowMemory() {
         view.onLowMemory();
     }
 
+    @Override public void onItemClick(AdapterView<?> parent, View view, int position, long id) {
+        final ArrayList<SceneUpdate> sceneUpdates = new ArrayList<>(1);
+        final String apiKey = "vector-tiles-tyHL4AY";
```

Comment:
Refactored a bit to at least not have same scene API setting 3 times in the code.